src/main/java/org/kohsuke/github/GHCodeScanningAlert.java

```diff
@@ -0,0 +1,258 @@
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+
+import java.io.IOException;
+import java.net.URL;
+import java.util.Date;
+
+/**
+ * Code scanning alert for a repository
+ *
+ * <a href="https://docs.github.com/en/rest/reference/code-scanning"></a>
+ */
+@SuppressFBWarnings(value = { "UUF_UNUSED_FIELD" }, justification = "JSON API")
+public class GHCodeScanningAlert extends GHObject {
+    @JsonIgnore
+    private GHRepository owner;
+    private long number;
+    private String html_url;
+    private GHCodeScanningAlertState state;
+    private GHUser dismissed_by;
+    private String dismissed_at;
+    private String dismissed_reason;
+    private Tool tool;
+    private Rule rule;
+    private GHCodeScanningAlertInstance most_recent_instance;
+    private String instances_url;
+
+    GHCodeScanningAlert wrap(GHRepository owner) {
+        this.owner = owner;
+        return wrap(owner.root);
+    }
+
+    GHCodeScanningAlert wrap(GitHub root) {
+        this.root = root;
+        if (owner != null) {
+            owner.wrap(root);
+        }
+        return this;
+    }
+
+    /**
+     * Id/number of the alert.
+     *
+     * @return the id/number
+     * @see #getId()
+     */
+    public long getNumber() {
+        return number;
+    }
+
+    /**
+     * Id/number of the alert.
+     *
+     * @return the id/number
+     * @see #getNumber()
+     */
+    @Override
+    public long getId() {
+        return getNumber();
+    }
+
+    /**
+     * State of alert
+     *
+     * @return the state
+     */
+    public GHCodeScanningAlertState getState() {
+        return state;
+    }
+
+    /**
+     * User that has dismissed the alert. Non-null when {@link #getState()} is <i>Dismissed</i>
+     *
+     * <p>
+     * Note: User object returned by code scanning GitHub API does not contain all fields. Use with caution
+     * </p>
+     *
+     * @return the user
+     */
+    public GHUser getDismissedBy() {
+        return dismissed_by;
+    }
+
+    /**
+     * Time when alert was dismissed. Non-null when {@link #getState()} is <i>Dismissed</i>
+     *
+     * @return the time
+     */
+    public Date getDismissedAt() {
+        return GitHubClient.parseDate(dismissed_at);
+    }
+
+    /**
+     * Reason provided for dismissing the alert.
+     *
+     * @return the reason
+     */
+    public String getDismissedReason() {
+        return dismissed_reason;
+    }
+
+    /**
+     * Code scanning tool that created this alert
+     *
+     * @return the tool
+     */
+    public Tool getTool() {
+        return tool;
+    }
+
+    /**
+     * Code scanning rule that was violated, causing the alert
+     *
+     * @return the rule
+     */
+    public Rule getRule() {
+        return rule;
+    }
+
+    /**
+     * Most recent instance of the alert
+     *
+     * @return most recent instance
+     */
+    public GHCodeScanningAlertInstance getMostRecentInstance() {
+        return most_recent_instance;
```

Comment:
@akashRindhe 
This is marked as not covered.  Please see if there's anything you can do to cover this method and any other methods that are not covered by tests.  Don't wear yourself out on it, but just take another look and see if any of them are easy cover.